compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -47,15 +55,20 @@ pub trait AllocBytes: Clone + fmt::Debug + Deref<Target = [u8]> + DerefMut<Targe
     /// - other pointers returned by this method, and
     /// - references returned from `deref()`, as long as there was no write.
     fn as_ptr(&self) -> *const u8;
+
+    /// Gets the allocation metadata.
+    fn get_mdata(&self) -> Self::ByteMetadata;
```

Comment:
What is this needed for?

Reply:
It is not! I left it over from an earlier attempt at making this work, I'll delete it